sdk/component/store/remote/postgresql/sqlc/migrations/20241209103854_findings_table.sql

```diff
@@ -0,0 +1,8 @@
+-- create "findings" table
+CREATE TABLE IF NOT EXISTS finding (
```

Comment:
Small thing, this is referred to as a "findings" table in comment and file name, but a "finding" (no s) table in the code.  Worth sticking with one or the other, either is fine but my preference would be "findings".

Reply:
Fixed.